apps/web/src/styles/realitease-fonts.css

```
@@ -36,53 +36,53 @@
-       url('https://pub-a3c452f3df0d40319f7c585253a4776c.r2.dev/fonts/reference%20fonts/NYTimes/NYTKarnak_Condensed.woff') format('woff'),
-       url('https://pub-a3c452f3df0d40319f7c585253a4776c.r2.dev/fonts/reference%20fonts/NYTimes/NYTKarnak_Condensed.ttf') format('truetype');
+  src: url('https://media.thereality.report/fonts/reference%20fonts/NYTimes/NYTKarnak_Condensed.woff2') format('woff2'),
+       url('https://media.thereality.report/fonts/reference%20fonts/NYTimes/NYTKarnak_Condensed.woff') format('woff'),
+       url('https://media.thereality.report/fonts/reference%20fonts/NYTimes/NYTKarnak_Condensed.ttf') format('truetype');
   font-weight: 700;
   font-style: normal;
   font-display: swap;
 }
 
 @font-face {
   font-family: 'nyt-karnak';
```

Stylelint (17.14.0)

[error] Expected no quotes around "nyt-karnak" (font-family-name-quotes)

(font-family-name-quotes)